paradise_papers_search/static/js/search.js

```diff
@@ -145,10 +148,29 @@
     }
 
 
+    fetchCount() {
+      $.getJSON(
+        this._search_api + 'count',
+        this._search_filters
+      )
+      .done(nodes => {
+        nodes = JSON.parse(nodes);
+        this._nodeSearchDataCount(nodes.response.data.count)
+      })
+
+      .fail(() => {
+        /*TODO Handle errors */
+        console.log("Fetch error");
+      })
+    }
+
+
+
     /** @todo init search filter and data */
     clear () {
       this._nodeSearchData([]);
       this._page(0);
+      this._nodeSearchDataCount(0);
```

Comment:
Here reset _nodeSearchDataCount to -1 again.

Reply:
💃